File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n = nums.size();
        set<int> st;
        for(auto it : nums) {
            st.insert(it);
        }
        int i = 0;
        for(auto it : st) {
            nums[i] = it;
            i++;
        }
        return st.size();
    }
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        LinkedHashSet<Integer> hs = new LinkedHashSet <Integer>();
        for(int i : nums)
        {
            hs.add(i);
        }
        int k = hs.size();
        int j = 0;
        for(int i : hs)
        {
            nums[j] = i;
            j++;
        }
       
        return k;
    }
}